recipe/meta.yaml

```diff
@@ -67,13 +69,15 @@ test:
     - $PYTHON -m pytest -v -m "not wheel" -rxXs -k "not (test_target_aligned_pixels or test_info_azure_unsigned)" tests  # [linux and build_platform == target_platform]
     - $PYTHON -m pytest -v -m "not wheel" -rxXs -k "not (test_target_aligned_pixels or test_outer_boundless_pixel_fidelity or test_info_azure_unsigned)" tests  # [linux and build_platform == target_platform]
     - $PYTHON -m pytest -v -m "not wheel" -rxXs -k "not (test_target_aligned_pixels or test_reproject_error_propagation or test_outer_boundless_pixel_fidelity)" tests  # [osx]
-    - python -m pytest -v -m "not wheel" -rxXs -k "not (test_hit_ovr or test_file_in_handler_with_vfs or test_files or test_parse_windows_path or test_copyfiles_same_dataset_another_name or test_context or test_target_aligned_pixels or test_decimated_no_use_overview or test_reproject_error_propagation)" tests  # [win]
+    # - python -m pytest -v -m "not wheel" -rxXs -k "not (test_decimated_no_use_overview or test_copyfiles_same_dataset_another_name)" tests  # [win]
```

Comment:
should the win tests be commented out?

Reply:
I plan on removing them altogether in my next commit to this PR as they are not necessary to exclude as these tests do not fail anymore for Windows. They are currently commented out to confirm that windows builds without any failing tests.

Reply:
no tests are running at all, though - you commented out the entire `python -m pytest` line. is that what you wanted?

Reply:
No, it isn't. Sorry about the oversight. I will revert the changes and restore the previous test configurations in my next commit to ensure the tests run as intended.